packages/dev-server-core/package.json

```diff
@@ -17,14 +17,17 @@
   "exports": {
     ".": {
       "import": "./index.mjs",
-      "require": "./dist/index.js"
+      "require": "./dist/index.js",
+      "types": "./index.d.ts"
```

Comment:
learned recently from Pascal that https://publint.dev/rules#exports_types_should_be_first

Reply:
Nice! Didn't know. :-)